packages/appeals-service-api/src/routes/api-docs.js

```diff
@@ -16,12 +16,14 @@ try {
 	logger.error(`problem generating api spec doc\n${err}`);
 }
 
-router.use(
-	'/',
-	swaggerUi.serve,
-	swaggerUi.setup(spec, {
-		explorer: true
-	})
-);
+var options = {
```

Comment:
I'd prefer const, but immaterial here.

Reply:
ugh must still have var ingrained in my brain, need to rebase anyway so will fix